src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -568,6 +570,46 @@ public static List<ClusterMetadata> tcmLoadRange(long min, long max)
         return afterLoad;
     }
 
+    /**
+     * This method exists due to the fact that we define a retry policy for TCM to follow, and then TCM ignores it and does no retries...
+     */
+    private static List<ClusterMetadata> reconstruct(long min, long max)
+    {
+        Epoch start = Epoch.create(min);
+        Epoch end = Epoch.create(max);
+        Retry.Deadline retryPolicyThatGetsIgnored = Retry.Deadline.retryIndefinitely(DatabaseDescriptor.getCmsAwaitTimeout().to(NANOSECONDS),
+                                                                      TCMMetrics.instance.fetchLogRetries);
+        Throwable lastError = null;
+        Backoff backoff = new Backoff.ExponentialBackoff(42, 200, SECONDS.toMillis(1), ThreadLocalRandom.current()::nextDouble);
```

Comment:
There's already a `Retry#Backoff` class that handles backoff that is used in conjunction with TCM retry functions. 

In the retrospect, I should have pushed back on addition of second backoff in CASSANDRA-19856. I asked for a follow-up patch, but it never happened, and now we have two diverging concepts.

Reply:
> There's already a Retry#Backoff class that handles backoff that is used in conjunction with TCM retry functions.

Reviewed and still would leave `Backoff.ExponentialBackoff` personally. 

* `Deadline` waits at most a fixed time - don't want that behavior
* `Backoff` is fixed wait, which I don't feel is a good behavior.  If things are failing we should actually *back off*.  Also, there are possibilities others are doing the same so you *should* have jitter to avoid DDOSing, but `Jitter` in TCM doesn't compose
* `Jitter` is just random sleep times

Reply:
I added a new `ExponentialBackoff` in retry to get the semantics I want (which delegates to `Backoff.ExponentialBackoff`)
I removed retry param from `getLocal` as no impl defines
I added retry on `RuntimeException` for paxos processor